### DISCUSSION TITLE
Whether Span information can be retrieved from org.slf4j.MDC after integration via agent.

### DISCUSSION THREAD

**Discussion by zhihaoHappyeveryday** [open-telemetry/opentelemetry-java-instrumentation#12035] [Q&A]
greet author :
Whether Span information can be retrieved from org.slf4j.MDC after integration via agent.

**Comment by trask** [comment]
hi! are you looking for `Span.current().getSpanContext()...`?

**Reply by zhihaoHappyeveryday** [reply]
> hi! are you looking for `Span.current().getSpanContext()...`?你好！您在寻找`Span.current().getSpanContext()...`吗？

dear author:
My scenario is to revamp dozens of old projects to integrate with opentelemetry.
Currently I'm trying to integrate a few core ones first.
Previously, the old project coded a lot of use of MDC to get link information for some push alerts.
I'm wondering if there's a way to use MDC to get link information without having to change a lot of previous code.
